dask_yarn/core.py

```diff
@@ -29,38 +52,81 @@ def make_remote_spec(environment, name='dask', queue='default', tags=None,
         The number of workers to initially start.
     worker_vcores : int, optional
         The number of virtual cores to allocate per worker.
-    worker_memory : int, optional
-        The ammount of memory in MB to allocate per worker.
+    worker_memory : str, optional
+        The ammount of memory to allocate per worker like '2GB' or '4096 MiB'.
```

Comment:
Could we support both strings and int (in MB)?

Reply:
We *could*, though I think it's a strange default.  Of course, it could be argued that it's a default choice that's made for us by Yarn.  Current convention in Dask is that if the user provides a number then that number corresponds to the number of bytes, not megabytes.

It's pretty obvious today what the user wanted, and we can automatically convert any number less than 1e6 to megabytes if that would be nicer.  I somewhat prefer the current convention because it is fully explicit, but I can see how it might be unpleasant to a user familiar with YARN conventions.  Happy to make the change.

Reply:
It's easier to add later than to remove. Fine as is.